src/libs/actions/Policy.js

```diff
@@ -1082,6 +1082,12 @@ function createWorkspace(ownerEmail = '', makeMeAdmin = false, policyName = '',
         if (transitionFromOldDot) {
             Navigation.dismissModal(); // Dismiss /transition route for OldDot to NewDot transitions
         }
+
+        // This is needed to redirect the user to workspace admin screen on the main router stack 
+        // https://github.com/Expensify/App/pull/19224
```

Comment:
```suggestion
        // To navigate the user to admin room
        // https://github.com/Expensify/App/issues
```

Wait for @NikkiWines before updating

Reply:
I would make the comment more generic to the behavior and less about the issue it's fixing. e.g. 

```
Get the reportID associated with the newly created #admins room and route the user to that chat
```

Reply:
@allroundexperts bump

Reply:
Yes it's fine

Reply:
Should remove the issue link @allroundexperts